### ISSUE TITLE
display bulk import file name as uploaded

### ISSUE THREAD

**Issue by TanyaStere42** [WildMeOrg/Wildbook#372]
**Context**
Wildbook currently does filename processing to remove characters that have caused problems
(see `function filesChangedSetFile` in `src/main/webapp/javascript/localUploader.js`)
We need to ensure that the original file name, which is stored but hidden, is what is displayed to the user and presented during export. This allows for easy reconciliation between files in system and external to the system.

**Current Behavior**
file `Caitlin_December_final.xlsx` currently displays as `CaitlinDecemberfinal.xlsx`

**Expected Behavior**
file `Caitlin_December_final.xlsx` currently displays as `Caitlin_December_final.xlsx`

**Acceptance Criteria**
Display is updated on the following pages:
- `src/main/java/org/ecocean/servlet/importer/StandardImport.java`
- `/src/main/webapp/imports.jsp`
- `/src/main/webapp/import.jsp`

**Comment by TanyaStere42** [comment]
Community link: https://community.wildme.org/t/underscores-dashes-no-longer-displaying-or-exporting-correctly/2721

**Comment by T4Cmaureen** [comment]
I'd just like to clarify that we also need the original image file name to display accurately on the match results page where it is currently displayed over top of the annotated target image and the annotated match candidate(s). Same criteria as above for exports:
Current Behavior
file Caitlin_December_final.xlsx currently displays as CaitlinDecemberfinal.xlsx

Expected Behavior
file Caitlin_December_final.xlsx currently displays as Caitlin_December_final.xlsx

Per this post on the Community forum: https://community.wildme.org/t/underscores-dashes-no-longer-displaying-or-exporting-correctly/2721?u=acwadmin1

**Comment by TanyaStere42** [comment]
Excellent! Thank you for catching that, I'll get the requirement updated to include that

**Comment by genzarchitect** [comment]
@TanyaStere42 I would like to give a try could you please assign it to me

**Comment by TanyaStere42** [comment]
Absolutely! If you find you need help with system setup or anything, checkout the discord (although we are heading into a long weekend)

**Comment by T4Cmaureen** [comment]
Hi all, any update on this issue? this remains a significant problem in that it doesn't allow users to reconcile their original datasets (pre-upload) with their exported data. Also causes issues deciphering the file names on match results where users intentionally insert location information to assist with confirmation, or not, of difficult match candidates. It was raised originally by an ACW user back in Aug and again in Nov '23. Hoping to see resolution of this soon. thanks

**Comment by Rodhlann** [comment]
Just so I understand, is the intent here to save the original filename instead of the processed filename? Or are we wanting to add an additional field for storing the original filename?

If the latter, why is it important to process the filename so that we are storing a different value than the original name?

**Comment by TanyaStere42** [comment]
I believe there were technical concerns around support for encoded characters, which caused the pre-processing to take place. @naknomum to confirm

At this point, the pre-processed version of the bulk import is used across the product, and rather than risk introduction of a regression somewhere, we just wanted to do the quick-and-easy of storing and displaying the original filename, but keeping the modified version throughout the pipeline

**Comment by naknomum** [comment]
i forget the backstory, but i do think we squash certain things (underscores is one) _during the file **upload**_. looking around, i am noticing that in `reviewDirectory.jsp` (which happens post-file-upload, but displays filenames) we use a [rather tricky reference to the original filenames via some session storage](https://github.com/WildMeOrg/Wildbook/blob/main/src/main/webapp/import/reviewDirectory.jsp#L109)! oh boy. so i suspect this is the data we might need to employ or this fix to reported missing files.

**Comment by JasonWildMe** [comment]
Some backstory on "why do we need to store the original filename somewhere else": users export data and like to cross-reference their data in Wildbook against their own internal data management systems. For some orgs, cross-referencing against the original filename is the easiest way to get data alignment between Wildbook and their system.
